runtime/solo/src/lib.rs

```diff
@@ -777,15 +856,16 @@ construct_runtime!(
 		Multisig: pallet_multisig = 17,
 		Utility: pallet_utility = 18,
 		Preimage: pallet_preimage = 19,
-		AwesomeAvatars: pallet_ajuna_awesome_avatars = 22,
-		Randomness: pallet_insecure_randomness_collective_flip = 23,
-		Nft: pallet_nfts = 24,
-		NftTransfer: pallet_ajuna_nft_transfer = 25,
-		NftStaking: pallet_ajuna_nft_staking = 26,
-		BattleMogs: pallet_ajuna_battle_mogs = 27,
-		AffiliatesAAA: pallet_ajuna_affiliates::<Instance1> = 28,
-		TournamentAAA: pallet_ajuna_tournament::<Instance1> = 29,
-		Migrations: pallet_migrations = 30,
+		// Migrations
+		Migrations: pallet_migrations = 20,
+		// SAGE - Extra
+		CasinoJamAffiliates: pallet_ajuna_affiliates::<Instance1> = 32,
+		CasinoJamTournament: pallet_ajuna_tournament::<Instance1> = 33,
+		// SAGE - Seasons
+		CasinoJamSeasons: pallet_ajuna_seasons::<Instance1> = 34,
+		// SAGE
+		CasinoJamSage: pallet_sage::<Instance1> = 40,
+		CasinoJamRandom: pallet_insecure_randomness_collective_flip = 41
```

Comment:
Just to be sure, there is testnet instance of the solo-net running anywhere? Just so we don't mess up something that is currently deployed.

Reply:
I do believe the testnet may have AAA running but it's not really used for anything as far as I'm aware